src/index.ts

```diff
@@ -151,61 +152,67 @@ async function runTests(
         await checkPackageJson(dirPath, typesVersions);
     }
 
+    const minVersion = maxVersion(
+        getMinimumTypeScriptVersionFromComment(indexText),
+        TypeScriptVersion.lowest) as TypeScriptVersion;
     if (onlyTestTsNext || tsLocal) {
         const tsVersion = tsLocal ? "local" : TypeScriptVersion.latest;
-        if (typesVersions.length === 0) {
```

Comment:
the only-next/only-local case is now much simpler because it doesn't have to search for the latest version inside a ts* directory